client/src/three/components/BuildingPreview.ts

```diff
@@ -86,6 +88,11 @@ export class BuildingPreview {
 
   public setBuildingPosition(position: THREE.Vector3) {
     if (this.previewBuilding) {
+      if (!this.currentHexHovered || !this.currentHexHovered.equals(position)) {
+        new Audio(dir + soundSelector.hoverClick).play();
```

Comment:
There was `useShovelSound()` hook in useUiSounds file, that plays two similar sounds one by one.
This sound was designed exactly for hex selection in building mode.

May be we can reuse it here?

Reply:
yeah let's do it

Reply:
Should I change it for the hex selection in the worldmap view as well?